pkg/attestation/crafter/api/attestation/v1/crafting_state.proto

```diff
@@ -150,12 +150,33 @@ message Attestation {
   // Signing options
   SigningOptions signing_options = 15;
 
+  // Runner environment in which the attestation was crafted
+  RunnerEnvironment runner_environment = 16;
+
   message SigningOptions {
     // TSA URL
     string timestamp_authority_url = 1;
   }
 }
 
+// The runner environment in which the attestation was crafted
+message RunnerEnvironment {
+     // Workflow file path that was used during build 
+  string workflow_file_path = 1;
+
+  // Runner environment name, i.e. github-hosted
+  string environment = 2;
+
+  // Whether the runner is authenticated, i.e. via the OIDC token
+  bool is_authenticated_runner = 3;
+
+  // Runner type
+  string type = 4;
```

Comment:
Is this typed by any chance? Shoul'd this be? workflowcontract.v1.CraftingSchema.Runner.RunnerType

Reply:
Good point, done.